sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -839,6 +839,19 @@ case class AlterTableSetLocationCommand(
 object DDLUtils {
   val HIVE_PROVIDER = "hive"
 
+  val METASTORE_GENERATED_PROPERTIES: Set[String] = Set(
```

Comment:
Just to check; does the latest hive metastore have the same set of these props here?

Reply:
I searched somewhere, but I think there is no determinate way to check this.

Reply:
Can we reuse `HiveClientImpl.HiveStatisticsProperties`? They are the hive specific properties we explicitly ignore.

Reply:
`HiveClientImpl.HiveStatisticsProperties` has been used at `HiveClientImpl.getTable()`, we needn't remove them again.

Reply:
I don't feel comfortable maintaining this list here. Is it possible to remove Hive specific properties in `HiveClientImpl`? So that the hive related stuff remains in hive related classes.